### PR TITLE
feat: update rkyv encoding & add solana base58 address verification

### PR DESCRIPTION
**Pull request by roberts-pumpurs**
## Description

- Update the rkyv encoding package which supports wasm32 in its encoding
- add base58 address verification mechanism to the voting verifier

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -59,7 +59,7 @@ serde_json = "1.0.89"
 service-registry = { version = "^1.0.0", path = "contracts/service-registry" }
 sha3 = { version = "0.10.8", default-features = false, features = [] }
 signature-verifier-api = { version = "^1.0.0", path = "packages/signature-verifier-api" }
-axelar-rkyv-encoding = { git = "https://github.com/eigerco/solana-axelar.git", branch = "main" }
```
Comment (roberts-pumpurs) [inline Cargo.toml:62]: Lets keep this PR open until the changes get synced to main

**Review by eloylp — APPROVED** [review]
lgtm

**Comment by roberts-pumpurs** [comment]
Lets merge this directly into `solana` as this branch also contains the base58 address verification (with the necessary fixes)

**Review by frenzox — APPROVED** [review]
lgtm!

**Pull request merged**